app/assets/js/app/modules/totalliser.js

```diff
@@ -0,0 +1,71 @@
+import domready from './domready'
+import {forEach} from 'lodash'
+import getTransitionEndEvent from '../helpers/transitionend'
+
+const transitionendEvent = getTransitionEndEvent()
+
+domready(() => {
+  const breakdownFields = document.querySelectorAll('.answer:not(.calculated) input')
+  const totalField = document.querySelectorAll('.calculated input')[0]
+
+  const onAnswerChanged = () => {
+    let newTotal = 0
+    let count = breakdownFields.length
+
+    let i
+    for (i = 0; i < count; i++) {
+      let percentage = breakdownFields[i].value
+
+      let value = parseInt(percentage)
+      if (isNaN(value)) {
+        continue
+      } else {
+        newTotal += value
+      }
+    }
+
+    updateTotal(newTotal)
+  }
+
+  const highlight = () => {
+    if (transitionendEvent) {
+      totalField.addEventListener(transitionendEvent, onTransitionHighlightEnd, true)
+    } else {
+      window.setTimeout(onTransitionHighlightEnd, 500)
+    }
+    totalField.classList.add('highlighted')
+  }
+
+  const removeHighlight = () => {
+    if (transitionendEvent) {
+      totalField.addEventListener(transitionendEvent, onTransitionHighlightEnd, true)
+    } else {
+      window.setTimeout(onTransitionHighlightEnd, 500)
+    }
+    totalField.classList.remove('highlighted')
+  }
+
+  const onTransitionHighlightEnd = (e) => {
+    totalField.removeEventListener(transitionendEvent, onTransitionHighlightEnd)
+  }
+
+  const updateTotal = (value) => {
+    totalField.value = value
+    if (value > 100 || value < 0) {
+      // bad
+      highlight()
+    } else if (value === 100) {
+      // good
+      removeHighlight()
+    } else if (value < 100 && value >= 0) {
+      // normal
+      highlight()
+    }
+  }
+
+  forEach(breakdownFields, (value) => {
+    value.addEventListener('change', onAnswerChanged)
```

Comment:
You should be able to add the event listener to a parent element of the fields as events bubble upwards through the DOM.

https://javascript.info/tutorial/bubbling-and-capturing